source/learning-wazuh/ssh-brute-force.rst

```diff
@@ -302,11 +302,11 @@ Paste that log record in a number of times.  On the 8th time, you should see a r
            Description: 'sshd: brute force trying to get access to the system.'
     **Alert to be generated.
 
-Press Control+C to exit ossec-logtest.  Then restart ``ossec-logtest`` but this time with the ``-v`` parameter for more detail.
+Press Control+C to exit wazuh-logtest.  Then restart ``wazuh-logtest`` but this time with the ``-v`` parameter for more detail.
 
 .. code-block:: console
 
-    [root@wazuh-manager centos]# /var/ossec/bin/ossec-logtest -v
+    [root@wazuh-manager centos]# /var/ossec/bin/wazuh-logtest -v
```

Comment:
`wazuh-logtest` doesn't support the `-v` parameter for verbosity. We have to take this into account to tune this section accordingly.